var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -68,20 +68,23 @@ class Amrex(CMakePackage, CudaPackage, ROCmPackage):
             description='Build particle classes')
     variant('plotfile_tools', default=False,
             description='Build plotfile_tools like fcompare')
-    variant('sundials', default=False,
-            description='Build AMReX with SUNDIALS support')
     variant('hdf5',  default=False,
             description='Enable HDF5-based I/O')
     variant('hypre', default=False,
             description='Enable Hypre interfaces')
     variant('petsc', default=False,
             description='Enable PETSc interfaces')
+    variant('sundials', default=False,
+            description='Enable SUNDIALS interfaces')
     variant('pic', default=False,
             description='Enable PIC')
 
     # Build dependencies
     depends_on('mpi', when='+mpi')
-    depends_on('sundials@4.0.0:4.1.0 +ARKODE +CVODE', when='@19.08: +sundials')
+    depends_on('sundials@4.0.0:4.1.0 +ARKODE +CVODE', when='@19.08:20.11 +sundials')
+    depends_on('sundials@5.7.0: +ARKODE +CVODE', when='@21.07: +sundials')
+    for arch in CudaPackage.cuda_arch_values:
+        depends_on(f'sundials@5.7.0: +ARKODE +CVODE +cuda cuda_arch={arch}', when=f'@21.07: +sundials +cuda cuda_arch={arch}')
```

Comment:
You might want to check with spack devs if they are still supporting Python 3.5. if I remember correctly, f-strings only work on Python>=3.6

Reply:
I see 3.5 in the unit tests as well as 2.7, so I will switch it.